### DISCUSSION TITLE
Build docker image for rook/ceph from source

### DISCUSSION THREAD

**Discussion by porwalameet** [rook/rook#12342] [General]
Hi Team,
I am not able to build a docker image from the rook/rook repo. I have tried instructions given in https://github.com/rook/rook/blob/master/INSTALL.md. I see the docker image build is 1.2GB, however the one published to quay.io is just around ~400MB for 1.11.7 tag

Command i followed:
`make -j2 IMAGES=ceph build.all`

Docker image:
```
REPOSITORY                                                 TAG              IMAGE ID       CREATED          SIZE
build-835db91d/ceph-arm64                                  latest           1f29ad1acef6   6 minutes ago    1.16GB
build-835db91d/ceph-amd64                                  latest           01cbb4b4272b   24 minutes ago   1.23GB
```

**Comment by travisn** [comment]
That looks correct. The images are quay are just compressed.

**Comment by porwalameet** [comment]
Thank @travisn . Also, i am interested to build only docker image. Is there a way I can avoid building CRDs and helm chart and build only docker image

**Reply by travisn** [reply]
`make build` doesn't currently support the option to skip building helm or the CRDs. PRs are welcome though if you would like to submit the change to add a new option.
